Form/Type/ColorSelectorType.php

```diff
@@ -16,6 +16,17 @@
 use Symfony\Component\OptionsResolver\OptionsResolver;
 use Symfony\Component\OptionsResolver\OptionsResolverInterface;
 
+@trigger_error(
+    'The '.__NAMESPACE__.'\ColorSelectorType class is deprecated since version 3.4 and will be removed in 4.0.'
+    .' Use '.__NAMESPACE__.'\ColorType instead.',
+    E_USER_DEPRECATED
+);
+
+/**
+ * NEXT_MAJOR: remove this class.
+ *
+ * @deprecated since version 3.4, to be removed in 4.0. Use ColorType instead
+ */
```

Comment:
I'm not sure we should deprecate this, since some people might need support for Internet Explorer or Safari, and shouldn't get this deprecation message. WDYT @sonata-project/contributors @jlamur?

Reply:
I agree, forgot about IE and Safari.

Reply:
Edge will support it, only IE is a problem: https://caniuse.com/#search=color%20input

Reply:
Oh right IE is dead in fact, correct? And Safari, well...

Reply:
The market share for old safari version is 1,5%...

IMO we could deprecate it because it is only a hint, that the developer shouldn't use it in the near feature.